sources/api/pluto/src/main.rs

```diff
@@ -383,3 +451,21 @@ mod test_pause_container_account {
         );
     }
 }
+
+mod test_get_resource_to_reserve_in_range{
```

Comment:
Missing `#[cfg(test)]`
```suggestion
#[cfg(test)]
mod test_get_resource_to_reserve_in_range{
```